docs/editors/vscode.md

```diff
@@ -156,7 +181,32 @@ supported `Run` view. When using Metals the debugger itself is
 [Bloop](https://scalacenter.github.io/bloop/), which is also responsible for
 starting the actual process.
 
-Users can begin the debugging session in two ways:
+Users can begin the debugging session in four ways:
```

Comment:
```suggestion
Users can begin the debugging session in tree ways:
```
?

Reply:
There are 4 ways mentioned:
- metals commands
- configuration
- code lenses
- test explorer

Reply:
Ah, you're right, missed the last one.

Reply:
I actually spent some time counting:P more complex task than one would imagine